File: server/aspnet/nvslonlineApi/nvslonlineApi/Models/Division.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace nvslonlineApi.Models
{
    public class Division
    {
        public int Id { get; set; }

        public string DivisionName { get; set; }

        public bool IsHidden { get; set; }
    }
}

File: client/angular2/NvslOnlineAngular2App/src/app/shared/models/division.ts
import { IDivision } from '../interfaces/interfaces';

export class Division implements IDivision{    
    
    constructor (public Id: number, public DivisionName: string, public IsHidden: boolean) {
        this.Id = Id;
        this.DivisionName = DivisionName;        
        this.IsHidden = IsHidden;
    } 
}